fluent-plugin-logdna.gemspec

```diff
@@ -5,10 +5,10 @@ Gem::Specification.new do |s|
   s.name        = 'fluent-plugin-logdna'
   s.version     = '0.3.1'
   s.date        = Date.today.to_s
-  s.summary     = 'LogDNA plugin for Fluentd'
-  s.description = 'Fluentd plugin for supplying output to LogDNA.'
-  s.authors     = ['Edwin Lai']
-  s.email       = 'edwin@logdna.com'
+  s.summary     = 'LogDNA Plugin for FluentD'
```

Comment:
Believe it's `Fluentd` with a lowercase 'd' on their website: https://www.fluentd.org/

Reply:
Done!